cli/command/formatter/history.go

```diff
@@ -79,21 +79,18 @@ func (c *historyContext) ID() string {
 }
 
 func (c *historyContext) CreatedAt() string {
-	var created string
-	created = units.HumanDuration(time.Now().UTC().Sub(time.Unix(int64(c.h.Created), 0)))
-	return created
+	return units.HumanDuration(time.Now().UTC().Sub(time.Unix(c.h.Created, 0)))
 }
 
 func (c *historyContext) CreatedSince() string {
-	var created string
-	created = units.HumanDuration(time.Now().UTC().Sub(time.Unix(int64(c.h.Created), 0)))
+	created := units.HumanDuration(time.Now().UTC().Sub(time.Unix(c.h.Created, 0)))
 	return created + " ago"
 }
 
 func (c *historyContext) CreatedBy() string {
 	createdBy := strings.Replace(c.h.CreatedBy, "\t", " ", -1)
 	if c.trunc {
-		createdBy = stringutils.Ellipsis(createdBy, 45)
+		return stringutils.Ellipsis(createdBy, 45)
```

Comment:
I don't have strong feelings about one of these forms versus the other, but I see the original as simpler because the function only returns from one place. If a linter is forcing an extra return, that seems a bit odd.